<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhysicsDescDialog</class>
 <widget class="QDialog" name="PhysicsDescDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="physicsGroup">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2">
       <widget class="QRadioButton" name="isStatic">
        <property name="text">
         <string>Static</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="geometry">
        <item>
         <property name="text">
          <string>Box</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sphere</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mesh</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="isDynamic">
        <property name="text">
         <string>Dynamic</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QRadioButton" name="isRegion">
        <property name="text">
         <string>Region</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="box">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Extents</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="VecEditWidget" name="boxExtents" native="true">
            <layout class="QGridLayout" name="gridLayout_5"/>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="sphere">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Radius</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="sphereRadius">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="mesh">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Mesh</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page">
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>10</y>
            <width>46</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Region</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VecEditWidget</class>
   <extends>QWidget</extends>
   <header>veceditwidget.h</header>
   <container>1</container>
   <slots>
    <signal>valueChanged()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>geometry</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PhysicsDescDialog</receiver>
   <slot>onGeometryChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PhysicsDescDialog</receiver>
   <slot>onAccept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onGeometryChanged(int)</slot>
  <slot>onAccept()</slot>
 </slots>
</ui>
